fvm/fvm_bench_test.go

```diff
@@ -76,6 +78,43 @@ func (account *TestBenchAccount) DeployContract(b *testing.B, blockExec TestBenc
 	require.Empty(b, computationResult.TransactionResults[0].ErrorMessage)
 }
 
+func (account *TestBenchAccount) AddArrayToStorage(b *testing.B, blockExec TestBenchBlockExecutor, list []string) {
```

Comment:
nit: could use some docs / explanation of WHY we want to test adding array to storage